src/excmds.ts

```diff
@@ -4304,6 +4306,9 @@ const KILL_STACK: Element[] = []
           - For example, use `bind ;jg hint -Jc .rc > .r > a` on google.com to generate hints only for clickable search results of a given query
         - -br deprecated, use `-qb` instead
         - -F [callback] - run a custom callback on the selected hint, e.g. `hint -JF e => {tri.excmds.tabopen("-b",e.href); e.remove()}`.
+        - -V create hints for invisible elements. By default, elements outside the viewport when calling :hint are not hinted, this includes them anyways.
+        - -! perform action immediately. This effectively selects every hinted element in sequence
+          - For example, `hint -!bf Comments` opens in background tabs all visible links whose text matches `Comments`
 
     Excepting the custom selector mode and background hint mode, each of these hint modes is available by default as `;<option character>`, so e.g. `;y` to yank a link's target; `;g<option character>` starts rapid hint mode for all modes where it makes sense, and some others.
```

Comment:
```suggestion
    Excepting the custom selector mode, background hint mode and the "immediate" modifier, each of these hint modes is available by default as `;<option character>`, so e.g. `;y` to yank a link's target; `;g<option character>` starts rapid hint mode for all modes where it makes sense, and some others.
```

We also need to add a `;V` bind following this convention (just plonk it in src/lib/config.ts alongside the others - should be obvious).

I'd be tempted to add `;![x]` binds too for most modes where it made sense.

Reply:
Added `;V` in f4560d8f. For the `-!` bindings I think we have to consider the risks, for example `;!b` would open all visible links in background pages, which might add up to quite a lot (and melt the user's machine).

Maybe in a follow-up PR we could have `!` keyboard shortcut (in hint mode) that does the same as `-!` does upfront, but giving the user a chance to review the hinted targets?